service/gcs/bridge/bridge.go

```diff
@@ -362,6 +363,9 @@ func (b *Bridge) createContainer(w ResponseWriter, r *Request) {
 		}
 		b.PublishNotification(notification)
 	}()
+
+	w.Write(response)
```

Comment:
Don't we want to write the response before the go routine? That way, we don't get an exit notification before the create call returns.

Reply:
Oh, right, my bad, you're right :)